Source/Core/Quaternion.js

```diff
@@ -1081,6 +1081,34 @@ define([
                 (Math.abs(left.z - right.z) <= epsilon) &&
                 (Math.abs(left.w - right.w) <= epsilon));
     };
+    
+    /**
+     * Computes the heading, pitch and roll from a quaternion (see http://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles )
+     *
+     * @param {Quaternion} quaternion The quaternion to retrieve heading,pitch and roll in radians.
+     * @returns {Object} an object that contains three attributs (in radians): roll,pitch and heading.
+     */
+    Quaternion.toHeadingPitchRoll = function(quaternion) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(quaternion)) {
+            throw new DeveloperError('quaternion is required');
+        }
+        //>>includeEnd('debug');
+
+        var result = {};
```

Comment:
We generally don't use implicit types like this throughout Cesium.  I think the way we would do this is to create a new type, `HeadingPitchRoll`, that has `heading`, `pitch`, and `roll` properties, and a `fromQuaternion` function.  We may also add this new type to existing Cesium APIs that take heading/pitch/roll, but that doesn't need to be part of this pull request.